src/main/java/core/basesyntax/service/FruitShopServiceImpl.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.service;
+
+import core.basesyntax.converter.DataConverter;
+import core.basesyntax.file.reader.FileReader;
+import core.basesyntax.file.writer.FileWriter;
+import core.basesyntax.handler.OperationHandler;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.report.ReportGenerator;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.List;
+
+public class FruitShopServiceImpl implements FruitShopService {
+    private final FileReader fileReader;
+    private final FileWriter fileWriter;
+    private final DataConverter dataConverter;
+    private final OperationStrategy operationStrategy;
+    private final ReportGenerator reportGenerator;
```

Comment:
According to the checklist, classes should not begin with an empty line. Please remove the empty line after the class declaration to follow the code style guidelines.